ts/components/ui/utils/hooks.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from "react";
+import Animated, {
+  Extrapolate,
+  interpolate,
+  interpolateColor,
+  useAnimatedStyle,
+  useDerivedValue,
+  useSharedValue,
+  withSpring
+} from "react-native-reanimated";
+import { IOColors, hexToRgba, useIOTheme } from "../../core/variables/IOColors";
+import {
+  IOScaleValues,
+  IOSpringValues
+} from "../../core/variables/IOAnimations";
+
+export const useListItemSpringAnimation = () => {
```

Comment:
I'm not a big fan of general purpose `hooks.ts` file. Instead, what do you think about a `ui/hooks` folder with a `useListItemSpringAnimation.ts` file in it?

Reply:
adressed in [25d682d](https://github.com/pagopa/io-app/pull/4710/commits/25d682d23923fbbf9128307b9230820e527f60ad)